src/main/java/org/redcrosswarriors/controller/ViewBloodRequestController.java

```diff
@@ -21,15 +21,12 @@
 
     @GetMapping("/viewRequests")
     @Secured("ROLE_USER")
-    public ResponseEntity<List<RequestInputDetails>> viewResults(Principal principal)
+    public ResponseEntity<Object> viewResults(Principal principal)
     {
```

Comment:
Please put the { on the same line as the method.

Reply:
Will make those changes